module3/labs/Week3/CreateSingleResource.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: Creating a Single Resource Lab
+---
+

```

Comment:
Great lab - I think at some point I want to start having students work through some of these challenges by writing tests first, and practicing some TDD; but not sure exactly when that should happen - would love to hear your thoughts!

Reply:
I think the flow is going to work really well to do this lesson code then tests, then later in the week do your related resources lesson code and then tests.

I hadn't thought of teaching a new topic with TDD, but after seeing your lesson I like the idea a lot and I think it will work very well here!